config.py

```diff
@@ -32,6 +32,7 @@ class Config(object):
     INVITE_EMAIL_FROM = 'enquiries@digitalmarketplace.service.gov.uk'
     INVITE_EMAIL_SUBJECT = 'Your Digital Marketplace invitation'
 
+    CREATE_USER_SUBJECT = 'Create your Digital Marketplace account'
```

Comment:
All content for these is here. If we want to move these, and maybe we do, I think it;s a later refactor.